addons/website_sale/views/templates.xml

```diff
@@ -1849,7 +1849,7 @@
                     <t t-foreach="product_images" t-as="product_image">
                         <div t-attf-class="carousel-item h-100#{' active' if product_image_first else ''}">
                             <div t-if="product_image._name == 'product.image' and product_image.embed_code" class="d-flex align-items-center justify-content-center h-100 embed-responsive embed-responsive-16by9">
-                                <t t-raw="product_image.embed_code"/>
+                                <t t-esc="product_image.embed_code"/>
```

Comment:
This fields probably need to be declared as html ;) 
![image](https://user-images.githubusercontent.com/4848797/114015954-9f35f180-986a-11eb-9c44-56195b288e8f.png)

Reply:
ndeed, I probably got confused between the embed_code from website_slides and the one from website_sales and missed one of the two.